CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Changed
 * Update .NET SDK to 6.0.200. [#2105](https://github.com/fsprojects/fantomas/pull/2105)
 
+### Fixed
```

Comment:
Really cool to see changelogs entries as part of the PRs :)

Reply:
Yes, well done to have spotted this.